CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -432,10 +533,23 @@ def checkRowForMismatch(row, columnDict, fixedColumnDict, baseColumnIdx=None, ig
 
     mismatchCell = wsOut.cell(row=getOutputCell(row[0], wsOut).row, column=1).offset(column=mismatchFlagIdx)
     if len(mismatchDict) > 0:
-        curMismatchFillStyle = LESSER_MISMATCH_FILL_STYLE_NAME
+        curMismatchFillStyle = None
+        mismatch_present = False
+        lesser_mismatch_present = False
```

Comment:
similarly here, looks like it should be possible to rewrite this without the new set of variables.